generator/config/stage/searchMeta.yaml

```diff
@@ -3,16 +3,34 @@ name: $searchMeta
 link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/searchMeta/'
 type:
     - stage
-encode: single
+encode: object
 description: |
     Returns different types of metadata result documents for the Atlas Search query against an Atlas collection.
     NOTE: $searchMeta is only available for MongoDB Atlas clusters running MongoDB v4.4.9 or higher, and is not available for self-managed deployments.
 arguments:
     -
-        name: meta
+        name: operator
+        mergeObject: true
         type:
-            - object
+            - searchOperator
+        description: |
+            Operator to search with.  You can provide a specific operator or use
+            the compound operator to run a compound query with multiple operators.
+    -
+        name: index
+        optional: true
+        type:
+            - string
+        description: |
+            Name of the Atlas Search index to use. If omitted, defaults to default.
```

Comment:
Should `default:` be used here?

Reply:
See https://github.com/mongodb/mongo-php-library/pull/1537#discussion_r1868405362